src/main/java/org/folio/edge/patron/PatronHandler.java

```diff
@@ -30,6 +36,7 @@
   public PatronHandler(SecureStore secureStore, PatronOkapiClientFactory ocf) {
     super(secureStore, ocf);
   }
+  private static final Logger logger = Logger.getLogger(Handler.class);
```

Comment:
This is a nitpick, but member variables should appear above constructors or functions